stream-chat-android-ui-common/src/main/kotlin/com/getstream/sdk/chat/viewmodel/messages/MessageListViewModel.kt

```diff
@@ -296,6 +296,7 @@ public class MessageListViewModel @JvmOverloads constructor(
         val reaction = Reaction().apply {
             messageId = message.id
             type = reactionType
+            score = 1
```

Comment:
Set score to "1" so that locally saved reaction is consistent with the one coming from the server (on the server, default score is "1")